File: SWIG-C/example_main.py
#!/usr/bin/pypy

import c_example

if __name__ == '__main__':
    a = c_example.Example('teszt-A')
    b = c_example.Example('teszt-B')

    print c_example.Get(a)
    print c_example.Get(b)
    print c_example.Get(a)
    print c_example.Get(a)
    print c_example.Get(a)
    print c_example.Get(a)
    print c_example.Get(b)

File: SWIG-C/example_main.cs
using System;

public class Example_main {
    public static void Main(string[] args)
    {
        var a = c_example.Example("Teszt-A");
        var b = c_example.Example("Teszt-B");

        Console.WriteLine(c_example.Get(a));
        Console.WriteLine(c_example.Get(b));
        Console.WriteLine(c_example.Get(a));
        Console.WriteLine(c_example.Get(a));
        Console.WriteLine(c_example.Get(a));
        Console.WriteLine(c_example.Get(a));
        Console.WriteLine(c_example.Get(b));
    }
}
